library.properties

```diff
@@ -0,0 +1,9 @@
+name=MemoryFree
+version=1.0
```

Comment:
Although this is valid, I recommend following semver:
https://semver.org/

Reply:
So int this case would you recommend 1.0.0 ? I think the behaviour is stable enough to define a public release.

Reply:
Sounds good to me.

Reply:
I have made this change